modules/serverless-creating-broker-admin.adoc

```diff
@@ -13,23 +13,42 @@ Brokers created due to annotation will not be removed if you remove the annotati
 ====
 
 .Prerequisites
-* Knative Eventing installed.
-* Cluster administrator permissions for {product-title}.
+* The {ServerlessOperatorName} and Knative Eventing are installed on your {product-title} cluster.
+* You have cluster administrator permissions for {product-title}.
 
 .Procedure
 
-* Annotate a namespace:
+* Annotate a namespace with the `eventing.knative.dev/injection: enabled` annotation:
 +
 [source,terminal]
 ----
-$ oc label namespace <namespace> eventing.knative.dev/injection=enabled
+$ oc label namespace <namespace> eventing.knative.dev/injection: enabled
```

Comment:
This change doesn't seem right, the original is correct for "oc label"

`oc label namespace <namespace> eventing.knative.dev/injection=enabled`